digits/templates/models/images/classification/new.html

```diff
@@ -647,12 +659,24 @@ <h4 style="display:inline-block;">Python Layers</h4>
                                 <div class="form-group{{mark_errors([form.custom_network])}}">
                                     {{form.custom_network.label}}
                                     {{form.custom_network.explanation(file='/models/images/classification/custom_network_explanation.html')}}
-                                    <div class="pull-right">
-                                        <a href=# class="btn btn-info" onClick="return visualizeNetwork();">Visualize</a>
-                                    </div>
-                                    {{form.custom_network(class='form-control', rows=10)}}
+                                    <div id="divCheckbox" style="display: none;">{{form.custom_network(class='form-control', rows=10)}}</div>
+                                    <a href=# class="btn btn-info" onClick="return visualizeNetwork();">Visualize</a>
                                 </div>
                         </div>
+                        <div id="ace-editor"></div>
+                        <script>
+                            var editor = ace.edit("ace-editor");
+                            editor.setTheme("ace/theme/monokai");
```

Comment:
I feel exactly the same: prefer the dark theme, but the Chrome theme is more in the color theme. Do we go for  Beauty or the Arts, your call? :)